### ISSUE TITLE
Research how to implement screen reader-friendly UI components

### ISSUE THREAD

**Issue by NonsenseSynapse** [NonsenseSynapse/draftivist#16]
This is a good starting point: https://bighack.org/5-most-annoying-website-features-i-face-as-a-blind-screen-reader-user-accessibility/

The result of this ticket should be a design document (Google Doc or something similar is probably good) with a breakdown of best practices for ensuring that our UI is screen reader-friendly. Some things to keep in mind:
* how big of a lift is it? Does it require additional libraries? Or is it as simple as adding additional tags to some components?
* how beneficial/detrimental is it to people using screen readers?
* are there any other easy wins we can make for accessibility since we are in the early phase of development and haven't committed too strongly to any given paradigm?

We don't necessarily have to do everything, but it will be helpful to know what we should start doing from the very beginning and what things we could potentially add after we have an initial version of the product done.
